### ISSUE TITLE
Bug: Can't Always Void COS Bills (SR) (Waiting on Bug to come up again)

### ISSUE THREAD

**Issue by Nadin-Nader** [cityofaustin/atd-data-tech#11521]
**Story: Incorrect Application Type is submitted and rejected:**
(Bill can't be voided on COS Payment)

1. Sys pays COS fees
2. We void the payment (Melissa w/ Finance)
3. Izzy voids the bill & reject the application (this is where the bug happens, Izzy hasn't seen it since Nov 14th, and has a list of the bills she still can't void. The system won't accept the void: Contact Administrator)

**Comment by vschandramouli** [comment]
Updated AC, moving to Backlog.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Analysis is in progress

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan here are the examples given:
Permit #2022 099868 EX               Invoice #7659414                              RSN 12971099    
Permit #2022 102799 EX               Invoice #7660881                              RSN 12974466    
Permit #2022 078086 RW             Invoice #7634790                              RSN 12945895    
Permit #2022 079837 RW             Invoice #7663064                              RSN 12947924
Permit #2022 105614 RW             Invoice #7664527                              RSN 12977689
Permit #2022 104924 RW             Invoice #7664541                              RSN 12976828
Permit #2022 104495 EX               Invoice #7663053                              RSN 12976343
Permit #2022 104474 EX               Invoice #7663048                              RSN 12976316
Permit #2022 104496 EX               Invoice #7663054                              RSN 12976344
Permit #2022 107506 RW             Invoice #7666828                              RSN 12979877
Permit #2022 109319 RW             Invoice #7669245                              RSN 12982118
Permit #2022 110649 EX               Invoice #7670020                              RSN 12983557
Permit #2021 062129 EX               Invoice #7674580                              RSN 12685780
Permit #2022 111809 EX               Invoice #7671695                              RSN 12985057
Permit #2022 115211 EX               Invoice #7676235                              RSN 12988861
Permit #2022 115281 EX               Invoice #7676358                              RSN 12988936
Permit #2022 115315 EX               Invoice #7676424                              RSN 12988980
Permit #2022 115287 EX               Invoice #7676390                              RSN 12988942
Permit #2022 115628 EX               Invoice #7676881                              RSN 12989307
Permit #2021 045615 EX               Invoice #7674248                              RSN 12666348
Permit #2022 101881 RW             Invoice #7659770                              RSN 12973396    
Permit #2021 073261 EX               Invoice #7677712                              RSN 12698909    
Permit #2022 152451 RW             Invoice #7721750                              RSN 13032451                    
Permit #2022 158710 RW             Invoice #7730756                              RSN 13039749    
Permit #2022 158861 EX               Invoice #7730992                              RSN 13039919    
Permit #2022 160469 RW             Invoice #7733302                              RSN 13041744
Permit #2021 1567887 EX            Invoice #7735425                              RSN 12795319
Permit #2022 159544 RW             Invoice #7731903                              RSN 13040722
Permit #2022 145484 RW             Invoice #7712827                              RSN 13024261
Permit #2021 1567887 EX            Invoice #7704538                              RSN 12795319
Permit #2022 159397 RW             Invoice #7737176                              RSN 13040548
Permit #2018 063853 RW             Invoice #7582862                              RSN 11919168

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan Here is what Kim and Kyle wrote for a response in the Testing Feedback 
TEAMs channel:

> [Wednesday 5:12 PM] Kampe, Kyle
>     Hi Nader, Nadin. Let's bring this one up again at the Friday meeting, and then once I get some more clarity on it I can meet with you and Sree individually on it next week.
> (1 liked)​[Yesterday 9:14 AM] Perez, Kimberly
>     Hey Nader, Nadin! This is what I posted in our DTS chat on 03/17 about this one:
> 
> 
> [3/17 3:12 PM] Perez, Kimberly
> 
> 
> So I can't find anything for making zero dollar balance payments.. The only thing I could find was that Mastec was able to make payments on two permits that the cost of service payments were voided, #2022-102799 EX and 2022-099868 EX. However, I think there might be another ticket for this.

**Comment by Nadin-Nader** [comment]
Update from Last Friday Meeting:
Isabel and Kyle were to meet on this to try to recreate. 
@TracyLinder may need to reassign card since Sree is out

**Comment by TracyLinder** [comment]
Kim gave clarification:


**Story: Incorrect Application Type is submitted and rejected:**
_(Bill can't be voided on COS Payment)_
1. Sys pays COS fees
2. We void the payment (Melissa w/ Finance)
3. Izzy voids the bill & reject the application (this is where the bug happens, Izzy hasn't seen it since Nov 14th, and has a list of the bills she still can't void. The system won't accept the void: Contact Administrator)

**Comment by Nadin-Nader** [comment]
Meeting Scheduled for 5/9/23 @ 10am with Sree, Izzy and Kim

**Comment by Nadin-Nader** [comment]
Meeting Rescheduled for 5/10/23 @130pm

**Comment by TracyLinder** [comment]
### Update / Mtg w/ ROW
- Sree is unable to recreate this issue now. 
- It's possible this issue was fixed when when #12009 went to PROD last week on 5/5.
- We don't know if this issue is fixed or not. We will put this in the ongoing pipeline to give time for new bugs to happen.

**Comment by Nadin-Nader** [comment]
@Nadin-Nader to follow up with Kyle to see if we can close or if bug has come up

**Comment by KyleErich** [comment]
After speaking to Izzy, this bug does still come up, but on the GF folder (for which we use the Austin Transportation sub type in order to to charge fees for TCP reviews). Example FolderRSN: 13142671

**Comment by TracyLinder** [comment]
### Update
ROW is still having consistent issues with this and works with Victor C. (CTM) regularly to get it resolved. They'd like us to talk to Victor to see if we can identify the root cause and fix it.

**Comment by TracyLinder** [comment]
Per ROW, this has been resolved by Victor and is safe to close:
